File: lab7/count_word.py
#!/usr/bin/python 
import sys
import re

wordCount = 0
matches = 0
pattern = sys.argv[1].lower()
#print pattern
#print "pattern to be matched is:", pattern
for line in sys.stdin:
    words = re.findall('[a-zA-Z]+',line,flags=0)
    for word in words:
        if word.lower() == pattern:
            matches +=1
    wordCount += len(words);
print sys.argv[1],"occurred" , matches,"times"

File: lab7/count_word.pl
#!/usr/bin/perl
use warnings;
	$pattern = "$ARGV[0]";
	$count = 0;
while($line = <STDIN>){
	@words = split/[^a-zA-Z]+/, $line;
	foreach $word (@words){
		if ($word =~ m/^$pattern$/i){
			$count++;
		}
	}
}

print "$pattern occurred $count times \n";
